### ISSUE TITLE
New strains update

### ISSUE THREAD

**Issue by keepgoing1995** [nextstrain/flu_frequencies#42]
Hi, @huddlej! Are you planning to update the dataset and include the new 2025 vaccine strains?

**Comment by huddlej** [comment]
@keepgoing1995 Yes, it's on the todo list for this month to update the flu_frequencies site.

[The new vaccines for H3N2](https://github.com/nextstrain/seasonal-flu/blob/76a2ddd741e188a600afd36d366eba7d70457e82/config/h3n2/vaccine.json#L93-L102) should be reflected in Nextstrain seasonal flu builds that we publish this week (usually posted by Thursday evening).

**Comment by keepgoing1995** [comment]
> @keepgoing1995 Yes, it's on the todo list for this month to update the flu_frequencies site.
> 
> [The new vaccines for H3N2](https://github.com/nextstrain/seasonal-flu/blob/76a2ddd741e188a600afd36d366eba7d70457e82/config/h3n2/vaccine.json#L93-L102) should be reflected in Nextstrain seasonal flu builds that we publish this week (usually posted by Thursday evening).

Great! Sorry for being unclear, I meant to ask if you plan to designate these strains as a new clade or subclade.

**Comment by keepgoing1995** [comment]
@huddlej  When you open the app, it says that it is temporarily paused. Is it undergoing an update or are there some technical issues with it?

**Comment by huddlej** [comment]
@keepgoing1995 Sorry about that! There is a technical issue with our web host and we're in the process of setting up an alternative host. We'll update the URL on the main repo page when we've finished migrating to the new host.

To answer your earlier question about new H3N2 clades, there haven't been any updates to the clade nomenclature for H3N2 within clade J.2 besides the recent addition of J.2.1 and J.2.2. We may eventually end up with a new clade name for the clade with the vaccine strains, but that change would likely happen after this February's vaccine composition meeting.

**Comment by keepgoing1995** [comment]
> [@keepgoing1995](https://github.com/keepgoing1995) Sorry about that! There is a technical issue with our web host and we're in the process of setting up an alternative host. We'll update the URL on the main repo page when we've finished migrating to the new host.
> 
> To answer your earlier question about new H3N2 clades, there haven't been any updates to the clade nomenclature for H3N2 within clade J.2 besides the recent addition of J.2.1 and J.2.2. We may eventually end up with a new clade name for the clade with the vaccine strains, but that change would likely happen after this February's vaccine composition meeting.

I apologise for opening/closing the issue several times. I didn't mean to spam here.

**Comment by keepgoing1995** [comment]
@huddlej, Has the application been successfully restored, or does it still need to be moved to another host?

**Comment by keepgoing1995** [comment]
@huddlej @rneher @ivan-aksamentov Has the application been successfully restored, or does it still need to be moved to another host?

**Comment by huddlej** [comment]
Hi @keepgoing1995, we're going to pause the flu_frequencies web application while we work on other approaches to frequency estimation and visualization. In the absence of the web interface to view frequencies, is it valuable to you to have access to occasionally updated frequency JSONs in this repository (e.g., [these USA-specific H3N2 HA clade frequencies](https://github.com/nextstrain/flu_frequencies/blob/master/web/public/data/pathogens/flu-h3n2-ha/geography/USA.json))?

If you were able to work from those files, we could update those periodically throughout the year as a resource until we have our new approach and web interface finalized.

**Comment by keepgoing1995** [comment]
> Hi [@keepgoing1995](https://github.com/keepgoing1995), we're going to pause the flu_frequencies web application while we work on other approaches to frequency estimation and visualization. In the absence of the web interface to view frequencies, is it valuable to you to have access to occasionally updated frequency JSONs in this repository (e.g., [these USA-specific H3N2 HA clade frequencies](https://github.com/nextstrain/flu_frequencies/blob/master/web/public/data/pathogens/flu-h3n2-ha/geography/USA.json))?
> 
> If you were able to work from those files, we could update those periodically throughout the year as a resource until we have our new approach and web interface finalized.

Sounds good. Thanks for the update!

**Comment by keepgoing1995** [comment]
> If you were able to work from those files, we could update those periodically throughout the year as a resource until we have our new approach and web interface finalized.

@huddlej  That would be great! Thanks for the update.

**Comment by keepgoing1995** [comment]
@huddlej Are the files you've shared in your previous comment the ones used as sources of sequenced case counts, or do you have other files for that?

**Comment by keepgoing1995** [comment]
@huddlej Are the files from your earlier comment the ones used as sources for the sequenced case numbers, or do you have different files for that?
